server/server.js

```diff
@@ -23,13 +61,42 @@ app.use(passport.session());
 
 /* Routes */
 app.use('/api/user', userRouter);
+app.use('/api/aws', awsRouter);
+app.use('/api/message', messageRouter);
+app.use('/api/vendor', vendorRouter);
+app.use('/api/search', searchRouter);
+app.use('/api/specialfeatures', specialFeaturesRouter);
+app.use('/api/vendortypes', vendorTypesRouter);
+
 
 // Serve static files
 app.use(express.static('build'));
 
 // App Set //
 const PORT = process.env.PORT || 5000;
 
+//Whenever someone connects this gets executed
+io.on('connection', (socket) => {
```

Comment:
Are we using this in the end for chat? Or are we just using the `/messages` endpoint?